css/style.css

```diff
@@ -53,8 +53,328 @@ p {
+    align-items: center;
+    margin: 20px 20px 20px 0;
+  }
+
+  .work li {
+    list-style-type: none;
+  }
+
+  .work-item {
+    grid-row: 1;
+  }
+
+  .works-title {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: var(--secondary-color);
+    text-transform: uppercase;
+  }
+
+  .work-role {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: var(--neutral-color);
+  }
+
+  .work-year {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: var(--neutral-color);
+  }
+
+  .btn {
+    padding: 12px;
+    color: var(--primary-color);
+    font-size: 17px;
+    line-height: 16px;
+    border-radius: 8px;
+    border: none;
+    background-color: var(--simple-color);
+  }
+
+  .btn:hover {
+    background-color: var(--primary-color);
+    color: #fff;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+    border-radius: 8px;
+  }
+
+  .btn:active {
+    background-color: #2230d2;
+    color: #fff;
+  }
+
+  .btn-disabled {
+    border: 1px solid #c1c7d0;
+  }
+
+  .btn-primary {
+    border: 1px solid var(--primary-color);
+    background-color: transparent;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .btn-secondary {
+    padding: 8px 12px;
+  }
+
+  .techs {
+    display: flex;
+    margin: 15px 24px 24px 0;
+    gap: 8px;
+  }
+
+  .techs li {
+    list-style-type: none;
+  }
+
+  .about-container {
+    display: flex;
+    background-color: #fff;
+    padding: 114px;
+    margin-top: 114px;
+    border-top-right-radius: 100px;
+  }
+
+  .about-heading {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .about-desc {
+    margin: 15px 0;
+    padding-right: 80px;
+  }
+
+  .about-container .social-icon {
+    margin: 16px 0 14px 0 !important;
+  }
+
+  .collapsible {
+    border-radius: 8px;
+    width: 100%;
+    height: 72px;
+    text-align: left;
+    padding: 12px 24px 0 26px;
+  }
+
+  .skils {
+    margin: 24px 0;
+  }
+
+  .skills-items {
+    list-style-type: none;
+  }
+
+  .skills-item {
+    width: 327px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .lang {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 72px;
+  }
+
+  .item {
+    width: 122px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 5%;
+    background-color: var(--general-color);
+    padding: 15px;
+    border-radius: 8px;
+    margin: -5px;
+  }
+
+  hr {
+    color: #7a869a;
+  }
+
+  .groups {
+    display: flex;
+    gap: 10px;
+  }
+
+  .contact-container {
+    background-color: var(--primary-color);
+    background-image: url(/img/contact_form_background_shape.png);
+    background-repeat: no-repeat;
+    padding: 142px 320px;
+    border-top-left-radius: 100px;
+    background-size: contain;
+    background-position: center;
+    top: -60px;
+    position: relative;
+  }
```

Comment:
- [ ] According to the requirements, You must replicate the design as much as possible (e.g., padding, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 
So Kindly remove the extra white space at the bottom of this section 👍

> 💡 I think you can fix it by removing `top: -60px` CSS property 

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/192979974-3ee1c762-8e46-4709-b82f-44c368f66926.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/192980188-1c51ea88-49fe-4e84-8572-3d52a7cb4ac9.png" width="500" /> |